### DISCUSSION TITLE
Panning on a zoomed in graph.

### DISCUSSION THREAD

**Discussion by bjhorrell** [jfree/jfreechart#342] [Q&A]
I have created an XYLine chart that I have enabled zooming.
My Xaxis goes from 0- 360 and the Yaxis goes from -90 to 90.
The user needs to be able to edit any point on the graph.
This will not fit on a standard monitor. 
I am looking for a way that once the user has zoomed in they can pan to different parts of the graph using the mouse. 
Any ideas on where to start for this?

Thanks in Advance.

**Comment by trashgod** [comment]
Some pan/zoom features and examples are cited [here](https://stackoverflow.com/a/44151032/230513). For selecting/editing, a [`ChartMouseListener`](https://www.jfree.org/jfreechart/javadoc/org/jfree/chart/ChartMouseListener.html), mentioned [here](https://stackoverflow.com/search?tab=votes&q=%5bjfreechart%5d%20ChartMouseListener),  will simplify converting model and view coordinates.
